docs/value.md

```diff
@@ -131,37 +130,38 @@ behavior:
   }
   ```
 
-
 @return {function} An optional teardown function. If provided, the teardown function
-will be called when the property is unbound after `stopListening()` is used to
-remove all bindings.
+  will be called when the property is unbound after `stopListening()` is used to
+  remove all bindings.
 
-The following `time` property increments every second.  Notice how a function
-is returned to clear the interval when the property is returned:
+  The following `time` property increments every second.  Notice how a function
+  is returned to clear the interval when the property is returned:
 
   ```js
-const Timer = DefineMap.extend( "Timer", {
-	time: {
-		value( prop ) {
-			prop.resolve( new Date() );
+  import {DefineMap} from "can";
 
-			const interval = setInterval( () => {
-				prop.resolve( new Date() );
-			}, 1000 );
+  const Timer = DefineMap.extend( "Timer", {
+    time: {
+      value( prop ) {
+        prop.resolve( new Date() );
 
-			return () => {
-				clearInterval( interval );
-			};
-		}
-	}
-} );
+        const interval = setInterval( () => {
+          prop.resolve( new Date() );
+        }, 1000 );
 
-const timer = new Timer();
-timer.on( "time", function( ev, newVal, oldVal ) {
-	console.log( newVal ); //-> logs a new date every second
-} );
+        return () => {
+          clearInterval( interval );
+        };
+      }
+    }
+  } );
+
+  const timer = new Timer();
+  timer.on( "time", function( ev, newVal, oldVal ) {
+    console.log( newVal.getSeconds() ); //-> logs a new date every second
+  } );
   ```
-  <!-- @codepen -->
+  @codepen
```

Comment:
This codepen should move (or create an additional) `console.log()` in the `setInterval`.  Then, it should show unbinding from `time` and that no more logs are printed.